packages/grid/_modules_/grid/models/colDef/colDef.ts

```diff
@@ -32,6 +32,10 @@ export interface ColDef {
    * @default 100
    */
   width?: number;
+  /**
+   * If set it idicates that a column has fluid width. Range [0 - 1].
```

Comment:
```suggestion
   * If set, it indicates that a column has fluid width. Range [0, 1].
```

Reply:
I think that we should match flex-grow API `[0, ∞]` to leverage past learnings from CSS: https://drafts.csswg.org/css-flexbox-1/#propdef-flex-grow
```suggestion
   * If set, it indicates that a column has fluid width. Range [0, ∞].
```

Reply:
We also need to update the docs